boom-app/src/types.ts

```diff
@@ -3,4 +3,13 @@ type CsvOutput = {
   data: Record<string, string>[]
 }
 
-export type { CsvOutput }
+/**
+ * Type to represent the JSON schema of an "overige object".
+ */
+type ObjectType = {
+  title: string
+  type: 'object'
+  properties: Record<string, { type: 'string' }>
+}
+
+export type { CsvOutput, ObjectType }
```

Comment:
Hier is niks mis mee, maar je kan ook dit doen:

export type CsvOutput = ....

export type ObjectType = ...